mvp-spec/security-signatures.md

```diff
@@ -0,0 +1,163 @@
+# Security and signatures
+
+## General signature rules
+
+Concerning the operator:
+- all **requests** are signed with:
+   - current timestamp
+   - protocol version ⚠️ not implemented yet ⚠️
+   - the "sender" domain that links to the public key
+   - the value of either the `origin` or the `referer` HTTP header
+     - note: it is mandatory for client websites to support the `referer` HTTP header.
+Websites that use a `Referrer-Policy` that prevent the `referer` header to be present would not work with PAF client.
```

Comment:
FAR: we should provide more detail on which calls and/or which call convention (redirect vs fetch) require origin and/or referer and the domains that are concerned (i.e the operator domain and the client node domain).
We should also provide detail of which level of detail is required in the referer (just the origin is enough right ?).
Optionally to provide a recap of which referer policies match the requirements (i think they all do, except for "no-referrer" and "same-origin". "strict-origin-when-cross-origin" if fine).

Reply:
> we should provide more detail on which calls and/or which call convention (redirect vs fetch) require origin and/or referer and the domains that are concerned (i.e the operator domain and the client node domain).
I believe this is more clear further down in the document, isn't it?

For the rest, I fully agree, thanks for the feedback.